loopd/swapclient_server.go

```diff
@@ -1154,6 +1160,21 @@ func validateLoopOutRequest(ctx context.Context, lnd lndclient.LightningClient,
 			errIncorrectChain, chainParams.Name)
 	}
 
+	// Check that the provided destination address is a supported
+	// address format.
+	switch sweepAddr.(type) {
+	case *btcutil.AddressWitnessScriptHash:
```

Comment:
nit: the `break` in Golang is implicit. So they aren't needed. And you can use an enumeration of values in a `case` statement, which I think is pretty handy too. And we usually add an extra line after a `case` body (except for the last). So this could look just like:

```
switch sweepAddr.(type) {
	case *btcutil.AddressWitnessScriptHash,
		*btcutil.AddressWitnessPubKeyHash,
		*btcutil.AddressScriptHash,
		*btcutil.AddressPubKeyHash,

	default:
		return 0, errInvalidAddress
	}
```